src/Nethermind/Nethermind.HealthChecks.Test/NodeHealthServiceTests.cs

```diff
@@ -366,5 +441,28 @@ private static string FormatMessages(IEnumerable<string> messages)
 
             return string.Empty;
         }
+
+        private class CustomRpcCapabilitiesProvider : IRpcCapabilitiesProvider
```

Comment:
That's just a detail, but class name could be more meaningful e.g. FixedRpc... or even TestRpc...

Reply:
I think it's some convention. `CustomSpecProvider` for example

Reply:
ok, that's fine